doc/concepts.rst

```diff
@@ -193,7 +193,8 @@ In addition, ``(action ...)`` fields support the following special variables:
 - ``exe:<path>`` is the same as ``<path>``, except when cross-compiling, in
   which case it will expand to ``<path>`` from the host build context
 - ``bin:<program>`` expands to a path to ``program``. If ``program``
-  is installed by a package in the workspace (see :ref:`install` stanzas),
+  is built by a package in the workspace (see :ref:`install`, :ref:`env` and
+  :ref:`executable` stanzas),
   the locally built binary will be used, otherwise it will be searched
   in the ``PATH`` of the current build context. Note that ``(run
```

Comment:
I'm confused, shouldn't `PATH` be used only for `exe` and the host build context?